pkgs/desktops/pantheon/apps/switchboard-plugs/about/fix-paths.patch

```diff
@@ -1,26 +1,33 @@
-diff --git a/src/Plug.vala b/src/Plug.vala
-index c32efcbe..0cdaeaca 100644
---- a/src/Plug.vala
-+++ b/src/Plug.vala
-@@ -178,7 +178,7 @@ public class About.Plug : Switchboard.Plug {
+From 8884ccf9591a7032f59293ae9a26dd97a8c448ce Mon Sep 17 00:00:00 2001
+From: Bobby Rong <rjl931189261@126.com>
+Date: Fri, 23 Jul 2021 14:51:04 +0800
+Subject: [PATCH 2/2] OperatingSystemView: fix paths
+
+---
+ src/Views/OperatingSystemView.vala | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/Views/OperatingSystemView.vala b/src/Views/OperatingSystemView.vala
+index 87bb3a4..dce29bf 100644
+--- a/src/Views/OperatingSystemView.vala
++++ b/src/Views/OperatingSystemView.vala
+@@ -99,7 +99,7 @@ public class About.OperatingSystemView : Gtk.Grid {
+         var bug_button = new Gtk.Button.with_label (_("Send Feedback"));
+ 
+         Gtk.Button? update_button = null;
+-        var appcenter_info = new GLib.DesktopAppInfo ("io.elementary.appcenter.desktop");
++        var appcenter_info = new GLib.DesktopAppInfo ("@appcenter@/bin/io.elementary.appcenter.desktop");
+         if (appcenter_info != null) {
+             update_button = new Gtk.Button.with_label (_("Check for Updates"));
+             update_button.clicked.connect (() => {
+@@ -147,7 +147,7 @@ public class About.OperatingSystemView : Gtk.Grid {
+         settings_restore_button.clicked.connect (settings_restore_clicked);
  
-         var bug_button = new Gtk.Button.with_label (_("Report a Problem"));
          bug_button.clicked.connect (() => {
 -            var appinfo = new GLib.DesktopAppInfo ("io.elementary.feedback.desktop");
 +            var appinfo = new GLib.DesktopAppInfo ("@elementary_feedback@/bin/io.elementary.feedback.desktop");
```

Comment:
I am not sure if this diff is good (as it is been here before this PR), I will look into this.

Reply:
Seems that changing this to `var appinfo = new GLib.DesktopAppInfo ("@elementary_feedback@/share/applications/io.elementary.feedback.desktop");` won't make the situation better even I have `pantheon.elementary-feedback` in my `environment.systemPackages`. I will drop this.